demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -272,7 +272,9 @@ def execute_update(self) -> bool:
             changed_files[(file_name, file_type)] = {
                 "description": get_file_description(packfile, file_type),
                 "is_new_file": packfile in self.added_files,
-                "fromversion": get_from_version_at_update_rn(packfile),
+                "fromversion": get_from_version_at_update_rn(packfile)
```

Comment:
Adding support for update-release-notes in case of pack_metadata changes results in a file without the "fromversion" field.